### ISSUE TITLE
List of other pages for the biomarker portal

### ISSUE THREAD

**Issue by ReneRanzinger** [glygener/glygen-issues#1237]
Go over all the pages in GlyGen that are reachable from:
* Main menu
* cards on home page
* Footer menu

And decide which of these pages are needed for the biomarker portal as well. Classify them into pages that are needed for the first version (home) and pages that can be added in a second version (supersearch).

After you are done assign the ticket to @sujeetvkulkarni. Based on your selection he will compile a list of web services that are needed beyond the biomarker specific services.

**Comment by sujeetvkulkarni** [comment]
@DaniallMasood please check https://glygen.ccrc.uga.edu/frontend/ for biomarker portal.
Please check which pages you want to keep/remove/add etc.
It currently uses glygen test apis and would require vpn.

**Comment by DaniallMasood** [comment]
@sujeetvkulkarni I would like to keep the pages on there (search, data, api,  and sparql). These look fine to me thank you

**Comment by DaniallMasood** [comment]
I am adding a check list of things that need to be removed/added/changed to the front end @sujeetvkulkarni 

- [x] Remove GlySpace Alliance
- [x] Changes to How To Cite and Our Mission (same as We Are GlyGen) (#1439 #1440 )
- [x] Remove Media page underneath About
- [x] remove version, featured video, statistics, news and other resources cards
- [x] Remove Tools pages
- [x] Change content in My Biomarker page (like My GlyGen pages) (#1441) 
- [x] Content for Help page (#1442) 
- [x] twitter, youtube, glygen wiki, pinterest links should be removed
- [x] change github link to https://github.com/clinical-biomarkers/biomarker-partnership
- [x] Data page should link to https://hivelab.biochemistry.gwu.edu/biomarker-partnership/data

Will provide content where necessary in another ticket. Please let me know if we can do these and when it can be expected. Also if you have any comments or suggestions on this as well.

**Comment by DaniallMasood** [comment]
@sujeetvkulkarni Raja and I would like these pages up by Friday. So if possible these changes could be made by Thursday morning?

**Comment by ReneRanzinger** [comment]
@DaniallMasood , @rajamazumder No, Sujeet is traveling today and we have the all hands meeting the rest of the week.

**Comment by DaniallMasood** [comment]
@ReneRanzinger @sujeetvkulkarni no problem then, that completely slipped my mind, sorry about that. Monday works for us as well if possible

**Comment by sujeetvkulkarni** [comment]
@DaniallMasood Like @ReneRanzinger said I was traveling to DC today and have whole day face to face meetings for rest of week. Will try to see what changes will be possible to fix by Monday.

**Comment by DaniallMasood** [comment]
@sujeetvkulkarni What i meant to say was that you can start this on Monday since you are busy this week sorry for the confusion. Whenever you get to this just let me know if you have questions about anything.

**Comment by sujeetvkulkarni** [comment]
@DaniallMasood ok sure, will get back to you if have any questions in this regard.

**Comment by DaniallMasood** [comment]
@sujeetvkulkarni is it possible to have a designation that this is a test website anywhere on the pages? maybe at the top? or have water mark on the pages?

**Comment by sujeetvkulkarni** [comment]
@DaniallMasood For API and SPARQL cards/links will there be some change or they will remain same.

**Comment by DaniallMasood** [comment]
@sujeetvkulkarni for now they will remain the same. We do not have SPARQL page yet but when we do we can link it. So if it is removable then it can removed for now.

For API card/link if @seankim658 has completed the swagger page we can link to that

**Comment by DaniallMasood** [comment]
@sujeetvkulkarni Also the date for the soft launch that was discussed in the biomarker partnership is June 28th, which is this Friday. Do you think this is doable by then?

**Comment by sujeetvkulkarni** [comment]
@DaniallMasood I am working on it currently and will finish it by then. You want it by Friday end of the day or Friday morning?

**Comment by DaniallMasood** [comment]
Friday morning if possible but if not then later on Friday is good too. Thank you @sujeetvkulkarni

**Comment by DaniallMasood** [comment]
@sujeetvkulkarni  what is the update on these tickets? thank you

**Comment by sujeetvkulkarni** [comment]
@DaniallMasood Most of the changes are done. I am going through final evaluations, will check in the changes, initiate the build and let you know once build is ready for testing today.

**Comment by sujeetvkulkarni** [comment]
@DaniallMasood added grant number to footer as per Rene's suggestion.

**Comment by sujeetvkulkarni** [comment]
@DaniallMasood 
Done. GitHub - 4777f226163f8a664b4ef1eb643ec417da9c65f0.
https://glygen.ccrc.uga.edu/frontend/
Please check and let me know.

**Comment by DaniallMasood** [comment]
@sujeetvkulkarni everything seems to be working from my end. Thank you so much fir getting this done.

**Comment by sujeetvkulkarni** [comment]
@DaniallMasood Great! I have added a TEST WEBSITE tag on the top of website, it will appear on all pages.

**Comment by DaniallMasood** [comment]
@sujeetvkulkarni thank you so much for all this. Just two things. Could we actually remove the sparql card and option in the drop down menu? Also the user setting that appears at the top still says GlyGen. Can that be changed to Biomarker Portal or removed?
![image](https://github.com/glygener/glygen-issues/assets/116293652/256aaf74-3ac2-4ac1-8d05-59ba7eb139e3)

**Comment by sujeetvkulkarni** [comment]
@DaniallMasood 
Removed sparql from header, footer and card.
Used Biomarker Partnership portal or Biomarker instead of GlyGen wherever necessary - please check.

**Comment by DaniallMasood** [comment]
looks good! thank you
